.github/workflows/pull_request.yml

```diff
@@ -55,7 +55,7 @@ jobs:
     runs-on: ubuntu-latest
     services:
       validator:
-        image: onsdigital/eq-questionnaire-validator:latest
+        image: onsdigital/eq-questionnaire-validator:remove-hard-coded-census-date
```

Comment:
revert before merge

Reply:
This has moved after [Update to use eq-questionnaire-validator repo](https://github.com/ONSdigital/eq-questionnaire-schemas/pull/4) has merged.

Reply:
It has changed since this [PR](https://github.com/ONSdigital/eq-questionnaire-schemas/pull/20) got merged. Not used anymore.